docs/install-macos-libomp.rst

```diff
@@ -0,0 +1,15 @@
+.. note::
+
+    If you don't have them, please first install:
+    `XCode <https://developer.apple.com/xcode/>`_, `Homebrew <https://brew.sh>`_, `LibOMP <https://formulae.brew.sh/formula/libomp>`_.
+    Once you have Homebrew, LibOMP can be installed via:
+
+    .. code-block:: bash
+
+        # brew install wget
+        wget https://raw.githubusercontent.com/Homebrew/homebrew-core/fb8323f2b170bd4ae97e1bac9bf3e2983af3fdb0/Formula/libomp.rb
+        brew uninstall libomp
```

Comment:
maybe have a comment here saying uninstall libomp if it's installed?

Reply:
uninstall when it isn't installed doesn't error in brew, so no downside

Reply:
I did get an error while experimenting the macOS CI though. Might be my mistake

Reply:
I checked and it does error:

```
brew uninstall abcdef
> Error: No available formula with the name "abcdef".
```

I think I didn't notice because I was entering line by line and the error didn't impact me, but it might if someone copy pastes the entire section.

Reply:
updated, thanks for checking!